vanilla-framework/vanilla-dashboard-theme
# Added white spinner icon and updated grey spinner

## Done

Added white spinner icon and updated grey spinner

## QA

- Check out the branch
- Run server ./run
- Go to examples http://0.0.0.0:8102/vanilla-dashboard-theme/examples/base/icons/icons-light/ and http://0.0.0.0:8102/vanilla-dashboard-theme/examples/base/icons/icons-dark/ 

## Details

Part of #11

## Screenshots

<img width="304" alt="screen shot 2017-07-19 at 14 52 12" src="https://user-images.githubusercontent.com/83575/28367613-e2d09c96-6c91-11e7-9e81-4f4ef9f5a142.png">
